src/Mvc/test/WebSites/BasicWebSite/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
```

Comment:
We don't need this right?

Reply:
I just added it to make sure that the site runs correctly when using F5 as i'm using the fully qualified url in code.

Reply:
Removed